src/psp-migration.md

```diff
@@ -0,0 +1,375 @@
+# How to migrate from Kubernetes PSPs to Kubewarden policies?
+Kubewarden can be used to replace all deprecated [PodSecurityPolicy](https://kubernetes.io/docs/concepts/security/pod-security-policy/). Different from the PSPs, Kubewarden has separated
+policies to achieve the same goal of different fields from the PSPs. The policies to cover each of the PSP configuration fields can be found below in the [mapping table](#mapping-kuberwarden-policies-to-psp-fields).
+
+To start deploying the Kubewarden policies to replace the PSPs, you need to install Kubewarden in your cluster. For that, please, refer to
+the [installation guide](./quick-start.md).
+
+Once you have our Kubewarden instance running, it's time to deploy some policies to replace PodSecurtyPolicy and keep your cluster secure. For that, you can start by listing
+the PSP in use. Let's suppose you have a PSP disabling privileged escalation, privileged containers, blocking pods running as root, forcing some user group, blocking host namespaces and allowing pod to use the port 443 only.
+
+Your PSP would be similar to this:
+
+```console
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: restricted
+spec:
+  allowPrivilegeEscalation: false
+  runAsUser:
+    rule: MustRunAsNonRoot
+  supplementalGroups:
+    rule: MustRunAs
+    ranges:
+      - min: 1000
+        max: 65535
+  privileged: false
+  hostNetwork: false
+  hostIPC: false
+  hostPID: false
+  hostPorts:
+    - min: 443
+      max: 443
+```
+
+Now, it's necessary to create the Kuberwarden policies to archive the same goal. It will be necessary 4 different policies. One for the user and group configuration, one for the host namespace, one for the privileged escalation and another for the privileged container configuration.
+
+Let's start by blocking container escalation. For that you can deploy a policy like this:
+
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: policies.kubewarden.io/v1alpha2
+kind: ClusterAdmissionPolicy
+metadata:
+  name: psp-allowprivilegeescalation
+spec:
+  module: registry://ghcr.io/kubewarden/policies/allow-privilege-escalation-psp:v0.1.11
+  rules:
+    - apiGroups:
+        - ""
+      apiVersions:
+        - v1
+      resources:
+        - pods
+      operations:
+        - CREATE
+        - UPDATE
+  mutating: false
+  settings:
+    default_allow_privilege_escalation: false
+EOF
+```
+
+Once this policy start to run, it will block pods trying to escalate its permissions:
```

Comment:
```suggestion
If you notice, we have specified `default_allow_privilege_escalation` to assume a value `false`. Once this policy starts running, it will restrict pods trying to run with more privileges than the parent container by default.
```